src/webhook_service/terraform/dev.tfvars

```diff
@@ -3,6 +3,7 @@ environment                                   = "dev"
 service_underscore                            = "webhook_service"
 service_hyphen                                = "webhook-service"
 dynamodb_table                                = "webhook"
+dynamodb_table_total_count                    = "webhook_total_count"
 enable_pitr                                   = false
 enable_deletion_protection_for_dynamodb_table = false
```

Comment:
Please enable the PITR for prod env, and enable the deletion protection for dev and prod env

Reply:
**Deletion protection is set to true in dev env.**
dev.tfvars
```
aws_region                                    = "us-east-1"
environment                                   = "dev"
service_underscore                            = "webhook_service"
service_hyphen                                = "webhook-service"
dynamodb_table                                = "webhook"
dynamodb_table_total_count                    = "webhook_total_count"
enable_pitr                                   = false
enable_deletion_protection_for_dynamodb_table = true
docker_host                                   = "unix:///Users/chungchihhan/.docker/run/docker.sock"
```
**Deletion protection and PITR are set to true in prod env.**
prod.tfvars
```
aws_region                                    = "ap-northeast-1"
environment                                   = "prod"
service_underscore                            = "webhook_service"
service_hyphen                                = "webhook-service"
dynamodb_table                                = "webhook"
dynamodb_table_total_count                    = "webhook_total_count"
enable_pitr                                   = true
enable_deletion_protection_for_dynamodb_table = true
docker_host                                   = "unix:///Users/chungchihhan/.docker/run/docker.sock"
```